src/Internal/QueryM/Pools.purs

```diff
@@ -97,8 +102,11 @@ getPubKeyHashDelegationsAndRewards pkh = do
   stringRep =
     ed25519KeyHashToBech32Unsafe "stake_vkh" $ unwrap $ unwrap pkh
 
+  -- byteHex :: String
+  -- byteHex = byteArrayToHex $ unwrap $ toBytes $ unwrap $ unwrap pkh
+
```

Comment:
Leftover?

Reply:
This one (and also line 83/84) is a bit of a mystery to me. The commented out versions produce a `No type class instance` error while the two step solutions (calculating first ScriptHash/Ed2551KeyHash and then serialize) work fine. afaics there should be no difference between the two..

Reply:
It's a mystery to me as well, seems like a purescript oddity. I'm going to ask about it in Slack, but deleting these comments for now as this is working